packages/ibm-products/src/components/FilterPanel/FilterPanel.mdx

```diff
@@ -11,6 +11,11 @@ import * as FilterPanelStories from './FilterPanel.stories';
 - [Example usage](#example-usage)
 - [Component API](#component-api)
 
+## Deprecated
```

Comment:
very minor, but can you change these headers to `Deprecation notice` to align with the `deprecationNotice` prop in `StoryDocsPage.js`?